docker/linea-sepolia/docker-compose.yml

```diff
@@ -121,15 +171,14 @@ services:
     volumes:
       - ./genesis.json:/genesis.json:ro
       - linea-sepolia-erigon:/home/erigon/.local/share/erigon/
-  
+
   # Nethermind node
   nethermind-node:
```

Comment:
Should we add them to Maru's config as followers, or what's the idea behind having these nodes? Only Besu node will peer as is.